Realm/Realm/Helpers/RealmObjectSerializer.cs

```diff
@@ -28,49 +28,68 @@
 namespace Realms.Serialization;
 
 #pragma warning disable SA1600 // Elements should be documented
-
 [EditorBrowsable(EditorBrowsableState.Never)]
-public abstract class RealmObjectSerializer : IBsonSerializer
+public static class RealmObjectSerializer
```

Comment:
I thought it was a little confusing that `RealmObjectSerializer` before was both a base class for serializers and contain a collection of serializers, so I've divided these two responsibilities in two different classes.